src/styles/styles.js

```diff
@@ -646,11 +647,13 @@ const styles = {
         paddingHorizontal: 11,
         borderWidth: 0,
         borderRadius: variables.componentBorderRadiusNormal,
+        ...expensiTextInputPlatformStyles,
     },
 
     expensiTextInputAndIconContainer: {
         zIndex: -1,
         flexDirection: 'row',
+        height: '100%',
```

Comment:
This could be breaking Multiline input. Could you test it and share a clip of that with me?

Reply:
@parasharrajat I tested Web Safari and App iOS and it was alright. I'll run screenshots once we settle the requested changes...